tools/cache_monitor.py

```
@@ -0,0 +1,441 @@
+                response_lines = [
+                    f"Unknown action: {action}. Valid actions: stats, health, report, cleanup, invalidate, warm"
+                ]
+
+            # Convert to TextContent objects
+            return [TextContent(type="text", text="\n".join(response_lines))]
+
+        except Exception as e:
+            logger.error(f"Cache monitor error: {e}")
+            return [TextContent(type="text", text=f"Cache monitor error: {str(e)}")]
+
+    async def _get_statistics(self, target: str) -> List[str]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)